status-whisper-usage-spec.md

```diff
@@ -59,6 +60,14 @@ Whisper's Proof Of Work algorithm is used to deter denial of service and various
 * proof-of-work requirement not larger than `0.002`
 * time-to-live not lower than `10` (in seconds)
 
+## Rate limiting
+
+In order to provide a very basic Denial-of-Service attack protection, there SHOULD be a rate limiting applied to leaf Whisper nodes, i.e. nodes not being controler within a cluster.
+
+Currently, the rate limiting is applied on peer IDs and IPs from which the packets come from. It is still in progress to decide on specific numbers. The Whisper nodes within a cluster SHOULD be whitelisted to avoid dropping connection when broadcasting messages between each other. When a limit is hit, the packet or the connection MUST be dropped (TBD).
+
+The rate limiting SHOULD be also applied on a topic level in order to prevent a peer to spam a particular channel. The specific limit per peer per topic is to be decided.
```

Comment:
```suggestion
The rate limiting SHOULD also be applied on the topic level in order to prevent a peer from spamming a particular channel. The specific limit per peer per topic is to be decided.
```